lib/routes/oauth.js

```diff
@@ -70,6 +70,102 @@ module.exports = (log, config, oauthdb) => {
         return oauthdb.getScopedKeyData(sessionToken, request.payload)
       }
     },
+    {
+      method: 'POST',
+      path: '/oauth/authorization',
+      options: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        validate: {
+          payload: {
+            client_id: validators.clientId.required(),
+            redirect_uri: Joi.string()
+              .max(256)
+              .uri({ scheme: ['https'] })
+              .required(),
+            scope: validators.scope.required(),
+            response_type: Joi.string()
+              .valid('code', 'token')
+              .default('code')
+              .required(),
```

Comment:
This is [not `required` in the OAuth server's version](https://github.com/mozilla/fxa-auth-server/blob/f6b303a40179a49c3c3ecc36ce56acdada50063e/fxa-oauth-server/lib/routes/authorization.js#L152), is that intentional?